re2c/src/parse/parser.ypp

```diff
@@ -159,7 +159,7 @@ void default_rule(CondList *clist, const Code * code)
 
 %token CLOSE
 %token CLOSESIZE
-%token CODE
+%token TOKEN_CODE
```

Comment:
Renamed this to `TOKEN_CODE` because it would produce `#define CODE ...` which in turn would collide with other declarations like `opt_t::CODE` or `OutputFragment::CODE`.